app/components/rdf-form-fields/validation-concept-scheme-selector.js

```diff
@@ -147,7 +149,38 @@ export default class ValidationConceptSchemeSelectorComponent extends InputField
       updateSimpleFormValue(this.storeOptions, option.subject);
     }
 
+    this.addgroupingTypeForValidationNode(
+      this.selected.subject,
+      this.storeOptions.store,
+      {
+        sourceGraph: this.storeOptions.sourceGraph,
+        metaGraph: this.storeOptions.metaGraph,
+      }
+    );
+
     this.hasBeenFocused = true;
     super.updateValidations();
   }
+
+  addgroupingTypeForValidationNode(validationType, store, graphs) {
+    const validationNode = store.any(
+      undefined,
+      RDF('type'),
+      validationType,
+      graphs.sourceGraph
+    );
+    const groupingType = getGroupingTypeForValidation(
+      validationType,
+      store,
+      graphs.metaGraph
+    );
+    store.addAll([
```

Comment:
I think we also need to remove the potential "old" value here?

Reply:
There are no remains in the code to clear up

Reply:
Where do those old values get removed then?

Reply:
@Windvis  I'm in a small conflict on what to do now.. I know I have to remove the grouping and the RDF type in this component. This assigning and updating was done by using the sh:path that is passed on to this component. Are we just removing this logic with the `path`as we know it needs to set and update the rdf- and the grouping type?

On the one side we lean to making this component ours by doing this.. instead of the method `updateSimpleValue` and on the other side we are still using this `path` thingy?
```js
      this.storeOptions.store.addAll([
        new Statement(
          this.storeOptions.sourceNode,
          FORM('grouping'),
          groupingType,
          this.storeOptions.sourceGraph
        ),
        new Statement(
          this.storeOptions.sourceNode,
          RDF('type'),
          option.subject,
          this.storeOptions.sourceGraph
        ),
      ]);
```

What should be a good solution? If we are not going to use the `sh:path` we can remove this.. but if we are still going to use it we will be adding/removing/updating the validation `type` with this code and the grouping in another way what makes it hard to know what is going on..
```js
const matches = triplesForPath(this.storeOptions, true).values;
      this.selected = this.options.find((opt) =>
        matches.find((m) => m.equals(opt.subject))
      );
```

Reply:
We agreed on removing the `path` stuff in the component. This is updated in the code to adding and removing the items by the predicate key (`rdf:type` & `form:grouping`)